src/EnergyPlus/CMakeLists.txt

```diff
@@ -241,6 +241,7 @@ set(SRC
     DualDuct.hh
     EMSManager.cc
     EMSManager.hh
+    EPVector.hh
```

Comment:
I'm not sure how other IDEs handle this, but without this the file doesn't appear in the Visual Studio project